docs/BUILD

```diff
@@ -1,34 +0,0 @@
-package(default_visibility = ["//visibility:public"])
```

Comment:
isn't this used by the docs site?

Reply:
It is, but on `main_v1` branch, so can delete safely here!